lm75bd/lm75bd.c

```diff
@@ -27,7 +27,24 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
   /* Implement this driver function */
+  error_code_t errCode;
   
+  uint8_t reg_addr = 0x0U;
+  errCode = i2cSendTo(LM75BD_OBC_I2C_ADDR, &reg_addr, 1U);
+  if (errCode != ERR_CODE_SUCCESS) return errCode;
+
+  uint8_t buff[2] = {0};
+  errCode = i2cReceiveFrom(LM75BD_OBC_I2C_ADDR, buff, 2U);
+  if (errCode != ERR_CODE_SUCCESS) return errCode;
```

Comment:
use `RETURN_IF_ERROR_CODE(function)` to ensure this error gets logged as well as to maintain consistency with our actual codebase